frontend/src/components/enroll/index.js

```diff
@@ -40,7 +40,7 @@ class Enroll extends Component {
         />
         <div className="container group">
           <div className="container col-sm-4 float-right mt-5">
-            <Markdown id="fontcss" className="mt-3 " source={this.state.text} />
+            <Markdown id="fontcss" className="text-left" source={this.state.text} />
```

Comment:
This looks great! It's a perfect, quick fix to make the text look more polished!